src/metalnx-web/src/main/java/com/emc/metalnx/modelattribute/user/UserForm.java

```diff
@@ -21,8 +19,6 @@ public class UserForm {
 	private String company;
 	private String department;
 	
-	private UserProfile userProfile;
```

Comment:
Noticed that we are removing "profile" stuff here, but that's not referenced in the commit message and I see some places where user profile stuff remains. Is that on purpose?

Reply:
Yes, the profile was retrieved from the irods-ext database so it is being removed.

I will add that to the commit messages and look for other profile code to remove.

Reply:
I removed a lot more dead code around templates, bookmarks, user profiles, and favorites.  Things seem to still work fine.